web/templates/settings/edit_custom_profile_field_form.hbs

```diff
@@ -1,53 +1,55 @@
 {{#with profile_field_info}}
-<form class="name-setting profile-field-form new-style" id="edit-custom-profile-field-form-{{id}}">
+<form class="name-setting profile-field-form new-style" id="edit-custom-profile-field-form-{{id}}" data-profile-field-id="{{id}}">
     <div class="input-group">
         <label for="name">{{t "Label" }}</label>
-        <input type="text" name="name" class="modal_text_input" value="{{ name }}" maxlength="40" />
+        <input type="text" name="name" id="id-name" class="modal_text_input prop-element" value="{{ name }}" maxlength="40" data-setting-widget-type="string" />
```

Comment:
I've used the `id-name` pattern here instead of a `id-custom-profile-field-name` type custom pattern. Please LMK if the former has any downsides I might not be aware of and I'll change it.

Reply:
`id-name` without a variable seems like it won't be globally unique in the DOM? It's a bug to every have duplicate IDs in an HTML page... so we should avoid anything that's so generic that it might be a duplicate.